cmd/multifile/multifile.go

```diff
@@ -70,7 +70,6 @@ func GetCommand() *cobra.Command {
 		Example: multifileExamples,
 	}
 
-	cmd.PersistentFlags().Int(optname.MaxConnPerHost, 40, "Maximum number of (global) concurrent connections per host")
```

Comment:
See my other comment- should this be a deprecation before pget errors saying "no such option"? We can don't need to use it even with a deprecation